skyportal/onboarding.py

```diff
@@ -105,7 +106,10 @@ def get_username(strategy, details, backend, uid, user=None, *args, **kwargs):
         raise Exception("PSA configuration error: `username` not properly captured.")
     storage = strategy.storage
 
-    existing_user = DBSession().query(User).filter(User.oauth_uid == uid).first()
+    with DBSession as session:
```

Comment:
The below should also be indented to be in the session I think?

Reply:
fixed